libraries/ms-drivers/src/adt7476a_fan_controller.c

```diff
@@ -101,3 +102,174 @@ StatusCode adt7476a_init(Adt7476aStorage *storage, Adt7476aSettings *settings) {
 
   return STATUS_CODE_OK;
 }
+
+#else
+#include <stdlib.h>
+
+#include "adt7476a.pb-c.h"
+#include "adt7476a_fan_controller.h"
+#include "adt7476a_fan_controller_defs.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "i2c.h"
+#include "interrupt.h"
+#include "soft_timer.h"
+#include "store.h"
+#include "stores.pb-c.h"
+
+static MxAdt7476aStore s_store = MX_ADT7476A_STORE__INIT;
+static uint32_t speed[NUM_ADT_PWM_PORTS] = { 0 };
+static bool status[NUM_ADT_PWM_PORTS] = { 0 };
+
+static void prv_update_speed() {
```

Comment:
I think we should just have one general export function; if prv_update_speed is called it could incorrectly set the state of the python since it doesn't update everything together.